scripts/test_schemas.sh

```diff
@@ -1,25 +1,61 @@
 #!/usr/bin/env bash
 
 docker pull onsdigital/eq-schema-validator
-validator=$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator)
+validator="$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator)"
 
 sleep 3
 
-echo "Testing Schemas"
+green="$(tput setaf 2)"
+red="$(tput setaf 1)"
+default="$(tput sgr0)"
+checks=4
+
+until [ "$checks" == 0 ]; do
+    response="$(curl -so /dev/null -w '%{http_code}' http://localhost:5001/status)"
+
+    if [ "$response" != "200" ]; then
+        echo "${red}---Error: Schema Validator Not Reachable---"
+        echo "HTTP Status: $response"
+        if [ "$checks"  != 1 ]; then
+            echo -e "Retrying...${default}\\n"
+            sleep 10
```

Comment:
10 seems a little long? essentially it will wait up to 40 seconds?

Reply:
@jonnyshaw89 It should only wait a max 30 seconds as on the 4th time it just exits. It was mainly 10 for when I was testing but I guess 5 seconds seems more appropriate.